README.md

```diff
@@ -66,6 +66,11 @@ Btw, nodist also works in your PowerShell, but you might first need to 'Unblock'
 ```
 
 ```
+> nodist 5
```

Comment:
Sorry for the delay -- this is a bad idea, because our shim has a different semver lib, that treats this as 5.0.0 instead of 5.x like npm's semver does.